components/bl-inputWithMask-component/component.json

```diff
@@ -0,0 +1,108 @@
+{
+  "id": "c_641f41b2f5632a22749fc992d3ca410e",
+  "name": "Input with Mask",
+  "description": "This component allows you add input with mask",
+  "showInToolbox": true,
+  "faIcon": "mask",
+  "mainJS": "dist/index.js",
+  "type": "custom",
+  "category": "Custom Components",
+  "properties": [
+    {
+      "type": "select",
+      "name": "maskType",
+      "label": "Mask Type",
+      "showInSettings": true,
+      "defaultValue": "Number",
+      "hasLogicHandler": true,
+      "handlerId": "maskTypeLogic",
+      "handlerLabel": "Mask Type Logic",
+      "options": [
+        {
+          "value": "Number",
```

Comment:
I assume values must be in lowercase because JS type of returns to us lowercase too.

Reply:
if the value is lowercase, eval(maskType) will be an error